copy-reports/copy-reports.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+OLD_POD=$(kubectl get pod -n kubecost -l app=cost-analyzer -o jsonpath='{.items[0].metadata.name}')
+NEW_POD=$(kubectl get pod -n kubecost -l app=aggregator -o jsonpath='{.items[0].metadata.name}')
+SOURCE_DIR="/var/configs"
+TARGET_DIR="/var/configs"
+
+FILES=(
+    "budgets.json"
+    "asset-reports.json"
+    "cloud-cost-reports.json"
+    "reports.json"
+    "alerts/alerts.json"
+)
+
+for FILE in "${FILES[@]}"; do
+    echo "Copying file: $FILE"
+
+    # Check the file in the old reports
+    kubectl exec -it -n kubecost $OLD_POD -c cost-model -- ls -lh "$SOURCE_DIR/$FILE"
```

Comment:
parameterized namespace

Reply:
do you do anything with this ls output? is this command necessary?

Reply:
this is probably not necessary, we know exactly which files need to be copied over.